### PR TITLE
Fix collector label addition issue

### PR DESCRIPTION
**Pull request by suresh-target**
Please refer the following issue to get more details on this PR, 

closes https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1426

### CODE REVIEW

File: charts/opentelemetry-kube-stack/examples/prometheus-otel/rendered/exporters/kube-scheduler/service.yaml [hunk 1]
```diff
@@ -7,7 +7,8 @@ metadata:
   labels:
     app: opentelemetry-kube-stack-kube-scheduler
     jobLabel: kube-scheduler
-    helm.sh/chart: opentelemetry-kube-stack-0.3.3
+    
+    helm.sh/chart: opentelemetry-kube-stack-0.3.4
```
Comment (jaronoff97) [inline charts/opentelemetry-kube-stack/examples/prometheus-otel/rendered/exporters/kube-scheduler/service.yaml:11]: why is this adding an extra space here?

Comment (suresh-target) [reply charts/opentelemetry-kube-stack/examples/prometheus-otel/rendered/exporters/kube-scheduler/service.yaml]: thanks @jaronoff97 for the review, this is fixed now.

**Comment by jaronoff97** [comment]
Could we also change one of the examples to include some extra labels in the `collector.labels` section? would love to have the examples capture this too :0

**Comment by jaronoff97** [comment]
thank you for your contribution! :bow:

**Pull request merged**